### PR TITLE
Add W Token Page

### PR DESCRIPTION
**Pull request by valentinoConti**
### Description

This PR adds a new section: **Wormhole Token Page**. 

Features the token metrics, the amount of transfers by chain, volume by chain, cross-chain transfers over time, top holders and top addresses by amount of transfers and by volume.

Responsive design implemented too.

### Screenshots

Screenshot 2024-09-06 at 08 53 19
Screenshot 2024-09-06 at 08 53 36
Screenshot 2024-09-06 at 08 53 47
Screenshot 2024-09-06 at 08 54 24

### CODE REVIEW

File: src/pages/Analytics/WToken/TopHolders.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                  </div>
+
+                  <div className="top-holders-table-item-row">
+                    <BlockchainIcon chainId={item.chain} network={environment.network} />
+                    <div>{chainIdToChain(item.chain)}</div>
+                  </div>
+
+                  <div className="top-holders-table-item-row">${formatNumber(+item.volume, 0)}</div>
+
+                  <div className="top-holders-table-item-row">??????</div>
```
Comment (giulianoconti) [inline src/pages/Analytics/WToken/TopHolders.tsx:61]: Unnecessary column?

File: src/pages/Analytics/WToken/TopHolders.tsx [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+                  <div className="top-holders-mobile-container-item">
+                    <div className="title">AMOUNT</div>
+                    <div className="content number">${formatNumber(+item.volume, 0)}</div>
+                  </div>
+
+                  <div className="top-holders-mobile-container-item">
+                    <div className="title">TXNS</div>
+                    <div className="content number">??????</div>
+                  </div>
```
Comment (giulianoconti) [inline src/pages/Analytics/WToken/TopHolders.tsx:113]: Unnecessary column?

File: src/pages/Analytics/WToken/TransfersOverTime.tsx [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+                  value={by}
+                />
+
+                <ToggleGroup
+                  ariaLabel="Select chart type"
+                  className="transfers-over-time-toggle-type"
+                  items={TYPE_CHART_LIST}
+                  onValueChange={value => setChartSelected(value as "area" | "bar")}
+                  value={chartSelected}
+                />
```
Comment (giulianoconti) [inline src/pages/Analytics/WToken/TransfersOverTime.tsx:131]: Design with little differences
pr

figma

**Review by giulianoconti — APPROVED** [review]
👏🏼👏🏼

File: src/pages/Analytics/WToken/TopHolders.tsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      </div>
+
+      {isDesktopDesign && (
+        <div className="top-holders-table">
+          <div className="top-holders-table-head">
+            <div className="top-holders-table-head-row">RANK</div>
+            <div className="top-holders-table-head-row">USER</div>
+            <div className="top-holders-table-head-row">CHAIN</div>
+            <div className="top-holders-table-head-row">AMOUNT</div>
+            <div className="top-holders-table-head-row">TXNS</div>
```
[the remark this answers is not in the document]
Comment (valentinoConti) [reply src/pages/Analytics/WToken/TopHolders.tsx:38]: It's needed just for it to use the space

image

File: src/pages/Analytics/WToken/TopHolders.tsx (hunk 1 above)
Comment (valentinoConti) [reply src/pages/Analytics/WToken/TopHolders.tsx:61]: Will be changed when the txns count data its available on the endpoint 🙏🏼

**Pull request merged**